### PR TITLE
DOCS-2756 Rewrite and reoganize SLO-based monitor page

### PR DESCRIPTION
**Pull request by urseberry**
### What does this PR do?
Update SLO-based monitor doc.

- Add section headers.
- Remove unneeded examples.
- Fix up prose.
- Add an explicit line saying that muting a monitor has no effect on the SLO calculation, as users frequently ask this question.

### Motivation
DOCS-2756

### Preview
https://docs-staging.datadoghq.com/uchen/mute-monitor-slo/monitors/service_level_objectives/monitor/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/service_level_objectives/monitor.md [hunk 1]
```diff
@@ -6,57 +6,75 @@ further_reading:
...
 
-### Set your SLO targets
+The time window you choose changes the available precision for your monitor-based SLOs:
+- 7-day and 30-day time windows allow up to two decimal places.
+- 90-day time windows allow up to three decimal places.
+
+In the details UI for the SLO, Datadog displays two decimal places for SLOs configured with 7-day and 30-day time windows, and three decimal places for SLOs configured with 90-day time windows.
 
-An SLO target is comprised of the target percentage and the time window. When you set a target for a monitor-based SLO the target percentage specifies what portion of the time the underlying monitor(s) of the SLO should be in an OK state, while the time window specifies the rolling time period over which the target should be tracked.
+A 99.999% target for a 7-day or 30-day time window results in an error budget of 6 seconds or 26 seconds, respectively. Monitors evaluate every minute, so the granularity of a monitor-based SLO is also 1 minute. Therefore, one alert would fully consume and overspend the 6 second or 26 second error budget in the previous example. In practice, teams cannot satisfy such small error budgets.
```
Comment (markazerdd) [inline content/en/monitors/service_level_objectives/monitor.md:59]: We should make it clearer that the point of this example is to illustrate why three decimal places don't make sense for 7-day or 30-day targets

File: content/en/monitors/service_level_objectives/monitor.md [hunk 2]
```diff
@@ -6,57 +6,75 @@ further_reading:
...
-### Identify this indicator
+## Example
 
-Here you can add contextual information about the purpose of the SLO, including any related information or resources in the description and tags you would like to associate with the SLO.
+You might be tracking the uptime of a physical device. You have already configured a metric monitor on `host:foo` using a custom metric. This monitor might also ping your on-call team if it's no longer reachable. To avoid burnout, you want to track how often this host is down. A monitor-based SLO using your custom metric can show you at a glance whether the amount of downtime for the `foo` host falls within your target range.
 
-## Overall status calculation
+## Status calculation
 
 {{< img src="monitors/service_level_objectives/overall_uptime_calculation.png" alt="overall uptime calculation"  >}}
```
Comment (markazerdd) [inline content/en/monitors/service_level_objectives/monitor.md:73]: Let's replace this image with https://a.cl.ly/X6ubgYo0

File: content/en/monitors/service_level_objectives/monitor.md [hunk 3]
```diff
@@ -6,57 +6,75 @@ further_reading:
...
+Choose a name and extended description for your SLO. Select any tags you would like to associate with your SLO.
 
-### Identify this indicator
+## Example
 
-Here you can add contextual information about the purpose of the SLO, including any related information or resources in the description and tags you would like to associate with the SLO.
+You might be tracking the uptime of a physical device. You have already configured a metric monitor on `host:foo` using a custom metric. This monitor might also ping your on-call team if it's no longer reachable. To avoid burnout, you want to track how often this host is down. A monitor-based SLO using your custom metric can show you at a glance whether the amount of downtime for the `foo` host falls within your target range.
```
Comment (markazerdd) [inline content/en/monitors/service_level_objectives/monitor.md:69]: This paragraph makes it sound like tagging your SLO is unique to custom metrics, which isn't true

File: content/en/monitors/service_level_objectives/monitor.md [hunk 4]
```diff
@@ -74,30 +92,38 @@ In certain cases, there is an exception to the status calculation for monitor-ba
...
+If you change any of these conditions to something other than their defaults, the overall status for a monitor-based SLO using one Synthetic test could appear to be better than the aggregated statuses of the Synthetic test's individual groups.
+
+For more information on Synthetic test alerting conditions, visit the Synthetic Monitoring [documentation][4].
+
+### Impact of manual and automatic monitor updates
 
-For more information on Synthetic test alerting conditions, visit the Synthetic Monitoring [documentation][7].
+When a monitor is resolved manually or as a result of the **_After x hours automatically resolve this monitor from a triggered state_** setting, SLO calculations do not change. If these are important tools for your workflow, consider cloning your monitor, removing auto-resolve settings and `@-notification` settings, and using the clone for your SLO. (**Note for Mark Azer:** This paragraph was in the original doc. I copied it, moved it here, and tried to reword it to be clearer, but I may have changed the meaning? Please review and clarify.)
```
Comment (markazerdd) [inline content/en/monitors/service_level_objectives/monitor.md:101]: This is still accurate

**Review by markazerdd — CHANGES_REQUESTED** [review]

File: content/en/monitors/service_level_objectives/monitor.md [hunk 5]
```diff
@@ -6,57 +6,73 @@ further_reading:
...
+Select **Save & Exit** to save your new SLO.
 
-## Overall status calculation
+## Status calculation
 
-{{< img src="monitors/service_level_objectives/overall_uptime_calculation.png" alt="overall uptime calculation"  >}}
+{{< img src="monitors/service_level_objectives/aggregate_slo.jpg" alt="SLO detail showing 99 percent green with 8 groups aggregated"  >}}
 
-The overall status can be considered as a percentage of the time where **all** monitors or **all** the calculated groups in a single multi-alert monitor are in the `OK` state. It is not the average of the aggregated monitors or the aggregated groups, respectively.
+Datadog calculates the overall status as the percentage of the time where **all** monitors or **all** the calculated groups in a single multi-alert monitor are in the `OK` state. Datadog does not calculate the average of the aggregated monitors or the aggregated groups.
```
Comment (markazerdd) [inline content/en/monitors/service_level_objectives/monitor.md:73]:
```suggestion
Datadog calculates the overall status as the percentage of the time where **all** monitors or **all** the calculated groups in a single multi-alert monitor are NOT in the `ALERT` state. Datadog does not calculate the average of the aggregated monitors or the aggregated groups. Monitor-based SLOs will also treat the `WARN` state as `OK`. This is because an SLO's definition requires a binary distinction between good and bad behavior, thus `WARN` is treated as good behavior since it is not severe enough to be considered bad behavior.
```

File: content/en/monitors/service_level_objectives/monitor.md [hunk 6]
```diff
@@ -74,30 +90,38 @@ In certain cases, there is an exception to the status calculation for monitor-ba
...
-For more information on Synthetic test alerting conditions, visit the Synthetic Monitoring [documentation][7].
+### Impact of manual and automatic monitor updates
+
+When a monitor is resolved manually or as a result of the **_After x hours automatically resolve this monitor from a triggered state_** setting, SLO calculations do not change. If these are important tools for your workflow, consider cloning your monitor, removing auto-resolve settings and `@-notification` settings, and using the clone for your SLO.
+
+Datadog recommends against using monitors with `Alert Recovery Threshold` and `Warning Recovery Threshold` as they can also affect your SLO calculations and do not allow you to cleanly differentiate between a SLI's good behavior and bad behavior.
+
+Muting a monitor has no effect on the SLO calculation.
+
+To exclude time periods from an SLO calculation, use the [SLO status corrections][5] feature.
```
Comment (markazerdd) [inline content/en/monitors/service_level_objectives/monitor.md:105]: We should add a new subsection here to talk about the impact of the NO DATA state. Ultimately what happens is: if the underlying monitor has notify_no_data set to false then NO DATA is treated as OK by the SLO, if the underlying monitor has notify_no_data set to true then NO DATA is treated as ALERT by the SLO.

**Review by markazerdd — CHANGES_REQUESTED** [review]

**Comment by urseberry** [comment]
This PR is now ready for docs team review.

File: content/en/monitors/service_level_objectives/monitor.md [hunk 7]
```diff
@@ -74,30 +92,43 @@ In certain cases, there is an exception to the status calculation for monitor-ba
 - Wait until a specified number of the groups are failing (default: 1)
 - Retry a specified number of times before a location's test is considered a failure (default: 0)
 
-By changing any of these conditions to something other than their defaults, the overall status for a monitor-based SLO using just that one Synthetic test could appear to be better than the aggregated statuses of the Synthetic test's individual groups. 
+If you change any of these conditions to something other than their defaults, the overall status for a monitor-based SLO using one Synthetic test could appear to be better than the aggregated statuses of the Synthetic test's individual groups.
+
+For more information on Synthetic test alerting conditions, see the Synthetic Monitoring [documentation][4].
```
Comment (urseberry) [inline content/en/monitors/service_level_objectives/monitor.md:97]:
```suggestion
For more information on Synthetic test alerting conditions, see [Synthetic Monitoring][4].
```

File: content/en/monitors/service_level_objectives/monitor.md [hunk 8]
```diff
@@ -74,30 +92,43 @@ In certain cases, there is an exception to the status calculation for monitor-ba
...
 
-Datadog recommends against using monitors with `Alert Recovery Threshold` and `Warning Recovery Threshold` as they can also affect your SLO calculations and do not allow you to cleanly differentiate between a SLI's good behavior and bad behavior.
+**Note:** SLOs based on Synthetic tests or Service Checks do not support SLO Replay.
 
-SLO calculations do not take into account when a monitor is resolved manually or as a result of the **_After x hours automatically resolve this monitor from a triggered state_** setting. If these are important tools for your workflow, consider cloning your monitor, removing auto-resolve settings and `@-notification`s, and using the clone for your SLO.
+## Other considerations
 
-Confirm you are using the preferred SLI type for your use case. Datadog supports monitor-based SLIs and metric-based SLIs as [described in the SLO metric documentation][6].
+Confirm you are using the preferred SLI type for your use case. Datadog supports monitor-based SLIs and metric-based SLIs as described in the [SLO metric documentation][3].
```
Comment (urseberry) [inline content/en/monitors/service_level_objectives/monitor.md:124]:
```suggestion
Confirm you are using the preferred SLI type for your use case. Datadog supports monitor-based SLIs and [metric-based][3] SLIs.
```

File: content/en/monitors/service_level_objectives/monitor.md [hunk 9]
```diff
@@ -6,57 +6,75 @@ further_reading:
...
-3. Select a single [multi-alert monitor][5] and select specific monitor groups (up to 20) to be included in SLO calculation using the **Calculate on selected groups** toggle.
+Select a **target** percentage, **time window**, and optional **warning** level.
 
-**Supported Monitor Types:**
+The target percentage specifies the portion of time the underlying monitor(s) of the SLO should not be in the ALERT state.
 
-- Metric Monitor Types (Metric, Integration, APM Metric, Anomaly, Forecast, Outlier)
-- Synthetic
-- Service Checks (open beta)
+The time window specifies the rolling time period over which the SLO runs its calculation.
```
Comment (apigirl) [inline content/en/monitors/service_level_objectives/monitor.md:46]:
```suggestion
The time window specifies the rolling period the SLO runs its calculation.
```

File: content/en/monitors/service_level_objectives/monitor.md [hunk 10]
```diff
@@ -6,57 +6,75 @@ further_reading:
...
 
-**Example:** You might be tracking the uptime of a physical device. You have already configured a metric monitor on `host:foo` using a custom metric. This monitor might also ping your on-call team if it's no longer reachable. To avoid burnout you want to track how often this host is down.
+Depending on the value of the SLI, the Datadog UI displays the SLO status in a different color:
+- While the SLI remains above the target, the UI displays the SLO status in green.
+- When the SLI falls below the target, the UI displays the SLO status in red.
+- If you included a warning level, and the SLI falls below the warning but above the target level, the UI displays the SLO status in yellow.
```
Comment (apigirl) [inline content/en/monitors/service_level_objectives/monitor.md:51]:
```suggestion
- If you included a warning level, and the SLI falls below the warning, but above the target level, the UI displays the SLO status in yellow.
```

File: content/en/monitors/service_level_objectives/monitor.md [hunk 11]
```diff
@@ -6,57 +6,75 @@ further_reading:
...
 
-### Set your SLO targets
+The time window you choose changes the available precision for your monitor-based SLOs:
+- 7-day and 30-day time windows allow up to two decimal places.
+- 90-day time windows allow up to three decimal places.
+
+In the details UI for the SLO, Datadog displays two decimal places for SLOs configured with 7-day and 30-day time windows, and three decimal places for SLOs configured with 90-day time windows.
```
Comment (apigirl) [inline content/en/monitors/service_level_objectives/monitor.md:57]:
```suggestion
In the details UI for the SLO, Datadog displays two decimal places for SLOs configured with 7-day and 30-day time windows and three decimal places for SLOs configured with 90-day time windows.
```

File: content/en/monitors/service_level_objectives/monitor.md [hunk 12]
```diff
@@ -6,57 +6,75 @@ further_reading:
...
+## Status calculation
 
-## Overall status calculation
+{{< img src="monitors/service_level_objectives/aggregate_slo.jpg" alt="SLO detail showing 99 percent green with 8 groups aggregated"  >}}
 
-{{< img src="monitors/service_level_objectives/overall_uptime_calculation.png" alt="overall uptime calculation"  >}}
+Datadog calculates the overall status as the percentage of the time where **all** monitors or **all** the calculated groups in a single multi-alert monitor are NOT in the `ALERT` state. Datadog does not calculate the average of the aggregated monitors or the aggregated groups. 
 
-The overall status can be considered as a percentage of the time where **all** monitors or **all** the calculated groups in a single multi-alert monitor are in the `OK` state. It is not the average of the aggregated monitors or the aggregated groups, respectively.
+Monitor-based SLOs treat the `WARN` state as `OK`. The definition of an SLO requires a binary distinction between good and bad behavior. SLO calculations treat `WARN` as good behavior, since `WARN` is not severe enough to indicate bad behavior.
```
Comment (apigirl) [inline content/en/monitors/service_level_objectives/monitor.md:75]:
```suggestion
Monitor-based SLOs treat the `WARN` state as `OK`. The definition of an SLO requires a binary distinction between good and bad behavior. SLO calculations treat `WARN` as good behavior since `WARN` is not severe enough to indicate bad behavior.
```

File: content/en/monitors/service_level_objectives/monitor.md [hunk 13]
```diff
@@ -74,30 +92,43 @@ In certain cases, there is an exception to the status calculation for monitor-ba
 - Wait until a specified number of the groups are failing (default: 1)
 - Retry a specified number of times before a location's test is considered a failure (default: 0)
 
-By changing any of these conditions to something other than their defaults, the overall status for a monitor-based SLO using just that one Synthetic test could appear to be better than the aggregated statuses of the Synthetic test's individual groups. 
+If you change any of these conditions to something other than their defaults, the overall status for a monitor-based SLO using one Synthetic test could appear to be better than the aggregated statuses of the Synthetic test's individual groups.
```
Comment (apigirl) [inline content/en/monitors/service_level_objectives/monitor.md:95]:
```suggestion
If you change any of these conditions to something other than their defaults, the overall status for a monitor-based SLO using one Synthetic test could appear better than the aggregated statuses of the Synthetic test's individual groups.
```

File: content/en/monitors/service_level_objectives/monitor.md [hunk 14]
```diff
@@ -74,30 +92,43 @@ In certain cases, there is an exception to the status calculation for monitor-ba
...
+
+For more information on Synthetic test alerting conditions, see [Synthetic Monitoring][4].
+
+### Impact of manual and automatic monitor updates
+
+When a monitor is resolved manually or as a result of the **_After x hours automatically resolve this monitor from a triggered state_** setting, SLO calculations do not change. If these are important tools for your workflow, consider cloning your monitor, removing auto-resolve settings and `@-notification` settings, and using the clone for your SLO.
+
+Datadog recommends against using monitors with `Alert Recovery Threshold` and `Warning Recovery Threshold` to underlie an SLO. These settings make it difficult to cleanly differentiate between an SLI's good behavior and bad behavior.
+
+Muting a monitor has no effect on the SLO calculation.
```
Comment (apigirl) [inline content/en/monitors/service_level_objectives/monitor.md:105]:
```suggestion
Muting a monitor does not affect the SLO calculation.
```

**Review by apigirl — APPROVED** [review]
Just some very small things, otherwise this looks great! Feel free to merge once you've accepted the suggestions and made sure the build is green.
